tests/api/core/content-manager/content-manager/history/history.test.api.ts

```diff
@@ -0,0 +1,442 @@
+        method: 'GET',
+        url: `/content-manager/history-versions/?contentType=${collectionTypeUid}&documentId=${collectionTypeDocumentId}&locale=fr`,
+      });
+
+      expect(collectionType.statusCode).toBe(200);
+      expect(collectionType.body.data).toHaveLength(2);
+      expect(collectionType.body.data[0].relatedDocumentId).toBe(collectionTypeDocumentId);
+      expect(collectionType.body.data[1].relatedDocumentId).toBe(collectionTypeDocumentId);
+      expect(collectionType.body.data[0].locale.code).toBe('fr');
+      expect(collectionType.body.data[1].locale.code).toBe('fr');
+      expect(collectionType.body.meta.pagination).toEqual({
+        page: 1,
+        pageSize: 20,
+        pageCount: 1,
+        total: 2,
+      });
+    });
+
+    test('A single type finds many versions in the default locale', async () => {
+      const singleType = await rq({
+        method: 'GET',
+        url: `/content-manager/history-versions/?contentType=${singleTypeUid}`,
+      });
+
+      expect(singleType.statusCode).toBe(200);
+      expect(singleType.body.data).toHaveLength(2);
+      expect(singleType.body.data[0].relatedDocumentId).toBe(singleTypeDocumentId);
+      expect(singleType.body.data[1].relatedDocumentId).toBe(singleTypeDocumentId);
+      expect(singleType.body.data[0].locale.code).toBe('en');
+      expect(singleType.body.data[1].locale.code).toBe('en');
+      expect(singleType.body.meta.pagination).toEqual({
+        page: 1,
+        pageSize: 20,
+        pageCount: 1,
+        total: 2,
+      });
+    });
+
+    test('A single type finds many versions in the provided locale', async () => {
+      const singleType = await rq({
+        method: 'GET',
+        url: `/content-manager/history-versions/?contentType=${singleTypeUid}&locale=fr`,
+      });
+
+      expect(singleType.statusCode).toBe(200);
+      expect(singleType.body.data).toHaveLength(2);
+      expect(singleType.body.data[0].relatedDocumentId).toBe(singleTypeDocumentId);
+      expect(singleType.body.data[1].relatedDocumentId).toBe(singleTypeDocumentId);
+      expect(singleType.body.data[0].locale.code).toBe('fr');
+      expect(singleType.body.data[1].locale.code).toBe('fr');
+      expect(singleType.body.meta.pagination).toEqual({
+        page: 1,
+        pageSize: 20,
+        pageCount: 1,
+        total: 2,
+      });
+    });
+
+    test('Applies pagination params', async () => {
+      const collectionType = await rq({
+        method: 'GET',
+        url: `/content-manager/history-versions/?contentType=${collectionTypeUid}&documentId=${collectionTypeDocumentId}&page=1&pageSize=1`,
+      });
+
+      expect(collectionType.body.data).toHaveLength(1);
+      expect(collectionType.body.meta.pagination).toEqual({
+        page: 1,
+        pageSize: 1,
+        pageCount: 2,
+        total: 2,
+      });
+    });
+  });
+
+  describe('Restore a history version', () => {
+    test('Throws with invalid body', async () => {
```

Comment:
What's the naming convention for API tests? Some are in the 3rd person, some aren't

Reply:
I've handled this on another branch as a follow up to this one with tests for restoring missing relations